src/components/EmptyState/EmptyState.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { forwardRef, useRef } from "react";
+import cx from "classnames";
+import { useMergeRefs } from "../../hooks";
+import VibeComponentProps from "../../types/VibeComponentProps";
+import VibeComponent from "../../types/VibeComponent";
+import { getTestId } from "../../tests/test-ids-utils";
+import { ComponentDefaultTestId } from "../../tests/constants";
+import styles from "./EmptyState.module.scss";
+import Flex from "../Flex/Flex";
+import Heading from "../Heading/Heading";
+import Text from "../Text/Text";
+import Button from "../Button/Button";
+
+export interface EmptyStateProps extends VibeComponentProps {
+  imgSrc: string;
```

Comment:
an empty state would also need support `SVG`, `Lottie` and maybe at some point different assets. why not keep this as a general node? this would also make `imgClassName` redundant